extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -89,7 +89,7 @@ public function  __construct( array $server, array $environment, array $wikiFact
 			$this->mWikiIdForced = true;
 
 			// differ CDN caching on X-Mw-Wiki-Id request header value
-			RequestContext::getMain()->getOutput()->addVaryHeader( 'X-Mw-Wiki-Id' );
+			RequestContext::getMain()->getOutput()->addVaryHeader( WebRequest::MW_WIKI_ID_HEADER );
```

Comment:
Now I started to wander why we don't have issues with other MW api endpoints we call from pandora services with this header...

Reply:
So this issues only arises for requests which use `action=ajax`, and it looks like our LinkSuggest request is the only one to use it so far...
```bash
james-s:pandora (master) jamessutterfield$ pwd
/Users/jamessutterfield/wikia/pandora
james-s:pandora (master) jamessutterfield$ grep -r 'action=ajax' *
Binary file service/discussion/build/classes/java/main/com/wikia/discussionservice/gateway/mediawiki/MediaWikiApi.class matches
service/discussion/src/main/java/com/wikia/discussionservice/gateway/mediawiki/MediaWikiApi.java:  @GET("/index.php?action=ajax&rs=getLinkSuggest&format=json&nsfilter=0&limit=20")
```